modules/cli/src/main/scala/scala/cli/commands/util/ScalacOptionsUtil.scala

```diff
@@ -0,0 +1,29 @@
+package scala.cli.commands.util
+
+import scala.build.options.{ScalacOpt, ShadowingSeq}
+import scala.cli.commands.ScalacOptions
+
+object ScalacOptionsUtil {
```

Comment:
I think these features are already implemented in `ShadowingSeq`, expect the only one - `get` function.

IMO in `ShadowingSeq` you should add the `get` function.

Reply:
done.